### ISSUE TITLE
posting buffers?

### ISSUE THREAD

**Issue by jimmywarting** [bitinn/node-fetch#136]
It's says that you support buffers

> body: empty        // request body. can be a string, **buffer**, readable stream

but when i send a simple buffer to httpbin it yields empty

So i tried something else: 
```javascript
    var stream = require('stream');
    var bufferStream = new stream.PassThrough();
    bufferStream.end(buffer);
```
i used bufferStream as body and send it to httpbin and that was successful

**Comment by bitinn** [comment]
We are likely missing a test case on this, shouldn't be too hard to add when I am available.

**Comment by bitinn** [comment]
fixed in v1.6.0
